apps/builder/app/shared/nano-states/nano-states.ts

```diff
@@ -237,12 +237,12 @@ export const selectedInstanceIntanceToTagStore = atom<
 >();
 
 /**
- * Whether or not the selected instance is rendered on canvas:
- * `true` — rendered;
- * `false` — not rendered;
- * `undefined` — selected instance unmounted and we don't know yet whether a new one will mount.
+ * pending means: previous selected instance unmounted,
+ * and we don't know yet whether a new one will mount
  **/
-export const selectedInstanceIsRenderedStore = atom<boolean | undefined>(false);
+export const selectedInstanceRenderStateStore = atom<
+  "mounted" | "not-mounted" | "pending"
```

Comment:
I would expect camel case notMounted or unmounted

Reply:
Changed to notMounted.

unmounted sounds wrong to me because it assumes it was mounted and then unmouted

Reply:
I think plain english unmounted doesn't imply it was previously mounted

Reply:
Even if not according to strict english rules, at least to me is sound like implies. Can be more people like me who may get confused